src/controller/CHIPDeviceController.h

```diff
@@ -460,13 +461,67 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController, public SessionEst
 
     static void OnSessionEstablishmentTimeoutCallback(System::Layer * aLayer, void * aAppState, System::Error aError);
 
-    CHIP_ERROR SendOperationalCertificateSigningRequestCommand(NodeId remoteDeviceId);
-    CHIP_ERROR OnOperationalCertificateSigningRequest(NodeId node, const ByteSpan & csr);
-    CHIP_ERROR SendOperationalCertificate(NodeId remoteDeviceId, const ByteSpan & opCertBuf, const ByteSpan & icaCertBuf);
-    CHIP_ERROR SendTrustedRootCertificate(NodeId remoteDeviceId, const ByteSpan & certBuf);
+    /* This function sends an OpCSR request to the device */
```

Comment:
nit: the "This function" prefix is very repetitive on all docs here. Consider a style of:

```cpp
/// Callback received when a commisionar receives the CSR from the device
...
/// Failure callbackwhen adding device operational certificates
```

most methods also do not need documentation. Things like `SendOperationalCertificate` obviously sends an operational certificate. It would be more useful to document argumeant lifetime for example (how long do the pointers in the passed in buffers need to live'?